File: 136-single-number/single-number.java
class Solution {
    public int singleNumber(int[] nums) {
        int n=nums.length;
        if (n==1){
            return nums[0];
        }
        int ans=nums[0];
        for(int i=1;i<n;i++){
            ans^=nums[i];
        }
        return ans;
        
    }
}

File: 136-single-number/single-number.py
class Solution(object):
    def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n=len(nums)
        if n==1:
            return nums[0]
        ans=nums[0]
        for i in range(1,n):
            ans^=nums[i]
        return ans